.circleci/config.yml

```diff
@@ -1,9 +1,43 @@
 version: 2.1
 
 orbs:
-  orb-tools: circleci/orb-tools@8.20.0
+  orb-tools: circleci/orb-tools@8.27.4
+  cli: circleci/circleci-cli@0.1.4
+
+jobs:
+  validate:
+    executor: cli/default
+    steps:
+      - checkout
+      - orb-tools/validate:
+          orb-path: src/@orb.yml
 
 workflows:
-  lint-validate:
+  publish:
     jobs:
       - orb-tools/lint
+      - validate
+      - orb-tools/publish-dev:
```

Comment:
Is there a difference between publish and publish-dev?

Reply:
`publish-dev` tags with dev:alpha and overrides this tag. Publish requires a semver which cannot be overwritten later